homeword-unit11-reactjs/main.js

```diff
@@ -0,0 +1,159 @@
+//Exercise 1
+
+// Viết 1 hàm trả về 1 array, nhận vào 2 tham số, tham số thứ 1 là ký tự cần lặp, tham số thứ 2 là số lần lặp
+// vd: đầu vào ('ahihi', 3)
+// Thì Kết quả là:
+// ['ahihi', 'ahihi', 'ahihi']
+
+// solution 1
+const parameter1 = (string, number) => {
+    const array = []
+    for (let i = 0; i < number; i++) {
+        array.push(string)
+    }
+    return array
+  }
+
+  // sulution 2
+  const parameter2 = (string, number) => {
+    const array = []
+    let i = 0 
+    while (i < number) {
+        array.push(string)
+      i++
+    }
+    return array
+  }
+
+//Exercise 2
+
+// Viết 1 hàm để đảo ngược 1 mảng, không được dùng hàm array.reverse()
+// Vd input:
+// ['a', 1, '9', 'apple']
+// Thì output phải là:
+// ['apple', '9', 1, 'a']
+
+const reverseTheArray = (array = []) => {
+    const array = []
+    for (let i = arr.length - 1; i >= 0; i--) {
+      result.push(input[i])
+    }
+    return array
+  }
+
+
+//Exercise 3
+
+// Xóa đi các giá trị được xem là sai
+// Vd intput:
+// [0, 1, false, 2, undefined, '', 3, null]
+// Thì output là:
+// [1, 2, 3]
+
+const deleteFailValues = (array = []) => {
+    const result = array.filter(character => character !== undefined && character !== null && character !== false && character !== '')
+    return result
```

Comment:
```suggestion
  return array.filter(character => character)
```

Và tab rất linh tinh